### DISCUSSION TITLE
Is there a way to tell what version you have from the radio?

### DISCUSSION THREAD

**Discussion by bobsbbq** [gdyuldin/x6100_gui#64] [General]
I have a serious SD car slot problem. I have about 4 or 5 SD cards that I flashed with two different versions. I was trying to figure out what was wrong. Now I know I have a bad SD card slot that it very intermittent. I have a card in there working right now. But I don't know which version it is v27, or v28, can I tell from the Radio what version it is using? With factory firmware there was a about section.
Thanks in advance. N5BSB

**Comment by gdyuldin** [comment]
Hi. Application shows a message with current version for some seconds right after boot.

**Reply by bobsbbq** [reply]
Ok thanks it was a little fast and I was missing it.
